src/main/kotlin/com/nextstep/stringcalculator/StringCalculator.kt

```diff
@@ -1,11 +1,11 @@
 package com.nextstep.stringcalculator
 
-class StringCalculator(input: String?) {
-    private val expression: Expression
-    init {
-        require(input != null && input.isNotBlank()) { "Input shouldn't be null or blank" }
-        this.expression = Expression(input)
-    }
+class StringCalculator {
 
-    fun calculate(): Long = expression.calculate()
+    companion object {
+        fun calculate(input: String?): Long {
+            require(input != null && input.isNotBlank()) { "Input shouldn't be null or blank" }
```

Comment:
String?.isNullOrBlank()를 활용해볼 수 있겠네요 :)
```suggestion
        fun calculate(input: String?): Long {
            require(input.isNullOrBlank()) { "Input shouldn't be null or blank" }
```